### DISCUSSION TITLE
Link to images stored in internal Budibase table are lost after updating to 2.13.17

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#12455] [Help]
I am using self host community edition. After updating from 2.8.10 to 2.13.17 i noticed that links to images that have been stored in a Budibase internal table I call images are broken. When I tried to upload the images again to a cell of that table then both the new and old image appear. Then i can delete the newly added image and keep the old old as the link is restored-fixed. 
Any idea why this is happening?

**Comment by ConorWebb96** [comment]
Hey @gmentzik,

I believe you were on a version of Budibase that didn't have signed urls. This is very likely the change that caused your images to break. There have been a lot of changes since then but this would be my best guess.

Happy to hear that a simple image upload fixes it.

**Reply by gmentzik** [reply]
What else could be broken because of this signed urls?
I can revert only to 2.8.10 as all versions I have tried since then have various issues. Latest major issue was breaking the DB Relationships functionalities. 2.8.10 is the latest for me that everything worked fine for my app.

**Reply by gmentzik** [reply]
Is there a way to try Budibase specific version with docker compose?
